build.gradle

```diff
@@ -22,6 +22,7 @@ buildscript {
 
     dependencies {
         classpath "com.android.tools.build:gradle:2.3.3"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.2.0"
```

Comment:
There are newer versions of the kotlin cradle plugin. See https://github.com/vestrel00/android-dagger-butterknife-mvp/pull/67/files#r164866268